libbeat/docs/dashboards.asciidoc

```diff
@@ -32,42 +32,54 @@ the _Beats Developer Guide_.
 ifndef::only-elasticsearch[]
 If you've configured the Logstash output, see
 <<load-dashboards-logstash>>.
-endif::[]
-
-ifdef::allplatforms[]
+endif::only-elasticsearch[]
 
 ifeval::["{requires-sudo}"=="yes"]
 
 include::../../libbeat/docs/shared-note-sudo.asciidoc[]
 
 endif::[]
 
-*deb and rpm:*
+ifdef::deb_os[]
```

Comment:
I think you can do `ifdef::deb_os,rpm_os[]` [example link](http://mrhaki.blogspot.com/2014/08/awesome-asciidoc-using-conditional.html)

Reply:
@ph I kind of grappled with this, too, because I don't like the repetition. I wanted to make sure that the conditional coding is granular enough to deal with situations where we might want to tag something for one OS but not the other. However, if there are two OS's that will always be supported together, then I can combine the commands. I think deb/rpm might be in this category, but I think Mac OS needs to be separate. WDYT? Linux + MacOS won't work together because Journalbeat (I believe) is linux-only, right? If you can help me identify the correct granularity for the conditions, I'll remove as much duplication as I can. My dream is to have a widget to switch between OS commands in the docs so things don't look so repetitive.

Reply:
@dedemorton  I think deb/rpm should be almost always the same. But yes you are right journalbeat is not on windows and not on mac.

Reply:
The command below are the same for mac, deb, rpm

Reply:
We could have a unix flavor? Looking a the doc, all of the mac, deb, rpm are the same.

Reply:
Looking the deb, rpm and macos commands they are pretty much the same, except that on mac you have `./` prefix to make sure we call the binaries in the current directory.